engine/src/test/java/org/operaton/bpm/engine/test/api/mgmt/metrics/JobExecutorMetricsTest.java

```diff
@@ -53,13 +53,17 @@ public class JobExecutorMetricsTest extends AbstractMetricsTest {
   public void saveJobExecutor() {
     processEngine = engineRule.getProcessEngine();
     defaultJobExecutor = processEngineConfiguration.getJobExecutor();
+
+    //Deletes all Metrics
```

Comment:
```suggestion
    // Deletes all Metrics
```
Personally, I would leave a space after opening the comment. You could leave that.